libs/core/shellbar/shellbar-branding/shellbar-branding.component.html

```diff
@@ -0,0 +1,20 @@
+<div
+    class="fd-shellbar__branding"
```

Comment:
You can apply the class and the role, tabindex, aria-label, etc. on the host. No need to have this div element.